compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs

```diff
@@ -198,10 +198,24 @@ fn main() {
         assert_eq!(intrinsics::align_of::<u16>() as u8, 2);
         assert_eq!(intrinsics::align_of_val(&a) as u8, intrinsics::align_of::<&str>() as u8);
 
+<<<<<<< HEAD
```

Comment:
It seems you missed those conflict delimiters.

Reply:
Oof. Indeed. And of course, this test is not check when running `x.py` yet. ^^'